chapters/annotations.tex

```diff
@@ -1747,13 +1747,12 @@ \subsubsection{Conversion rules}\label{conversion-rules}
    {"y0=0", "k1=0", "k2=0"}, {"y0=%y0%", "k1=%y0%*%k1%", "k2=%y0%*%k2%"},
    true);
 convertClass(
-   "Modelica.Electrical.Machines.BasicMachines.QuasiStationaryDCMachines",
-   "Modelica.Electrical.Machines.BasicMachines.QuasiStaticDCMachines");
-convertElement("Modelica.Electrical.Machines.Interfaces.PartialBasicDCMachine",
-               "quasiStationary", "quasiStatic");
-convertElement("Modelica.Electrical.Machines.BasicMachines." +
-                "QuasiStationaryDCMachines.DC_ElectricalExcited",
-               "quasiStationary", "quasiStatic");
+   "My.Library.Pakkage",
```

Comment:
Should probably be `Package` (spelling), and the same throughout the rest of the changes?

Reply:
It was a deliberate misspelling, since it was intended to be converted.
But I could replace it with something else to make that clearer. Like "BadPackage"

Reply:
Now done.

Reply:
Sorry, guess I should have read a bit closer and with more context. Regardless, I think this is an improvement!